pyproject.toml

```diff
@@ -26,7 +26,7 @@ nornir-napalm = "^0.1.1"
 nornir-jinja2 = "^0.1.1"
 nornir-netmiko = "^0.1.1"
 pynautobot = "^1.0.1"
-netutils = "^0.1.2"
+netutils = "^0.2.1"
```

Comment:
I think I may get confused by poetry's syntax sometimes, is this better served with a `~0.X` syntax? My intention of the original was it should have just picked up for any user installing and this should not have been needed, but I suspect that is not the case. If so, should probably fix the original issue.

Reply:
If just any version should be installed, based on the time of install then `*` would handle this.

Reply:
It is not any for sure, it is above 1.2 that they would need, but I am not sure what that means for poetry spec.

Reply:
Updated to use ^0 per the discussion below.